task/thermal/src/main.rs

```diff
@@ -15,8 +15,8 @@
 #[cfg_attr(target_board = "gimlet-b", path = "bsp/gimlet_bc.rs")]
 #[cfg_attr(target_board = "gimlet-c", path = "bsp/gimlet_bc.rs")]
 #[cfg_attr(
-    any(target_board = "sidecar-a", target_board = "sidecar-b"),
-    path = "bsp/sidecar_ab.rs"
+    any(target_board = "sidecar-b", target_board = "sidecar-c"),
```

Comment:
Nit: we should make the Sidecar and Gimlet `cfg_attr` consistent here; either
```rust
#[cfg_attr(
    any(target_board = "gimlet-b", target_board = "gimlet-c"),
    path = "bsp/gimlet_bc.rs"
)]
#[cfg_attr(
    any(target_board = "sidecar-b", target_board = "sidecar-c"),
    path = "bsp/sidecar_bc.rs"
)]
```
or
```rust
#[cfg_attr(target_board = "gimlet-b", path = "bsp/gimlet_bc.rs")]
#[cfg_attr(target_board = "gimlet-c", path = "bsp/gimlet_bc.rs")]
#[cfg_attr(target_board = "sidecar-b", path = "bsp/sidecar_bc.rs")]
#[cfg_attr(target_board = "sidecar-c", path = "bsp/sidecar_bc.rs")]
```

I like the upper option more, but don't feel too strongly about it.